### DISCUSSION TITLE
Add multiple stops for routes

### DISCUSSION THREAD

**Discussion by Raycock2** [organicmaps/organicmaps#2473] [Ideas]
Adding multiple stops for a route. This would be good is someone is on a long route and needed to stop for food, which may be off the route.

**Comment by davidcollini** [comment]
This would also be great for finding gas stations or charging stations on a route

**Comment by vachan-maker** [comment]
Hm..
Isn't this already possible in the app?

**Reply by davidcollini** [reply]
It seems that it's possible to search for ex: a gas station when going on a route, but this doesn't give the gas station that would add the least amount of time to the overall route

**Comment by biodranik** [comment]
@Raycock2 You already can add stops when building a route. Select any point on the map and press "Add Stop" on the bottom bar.

**Comment by Raycock2** [comment]
Just found it. Thanks for the explanation on how to get to it.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by randomizeduser** [comment]
I created a similar discussion. It's about showing you distance to next POI while navigating, but not necessarily your next waypoint:

#2540
